pkg/log/sqlitelogger/sqlite_logger.go

```diff
@@ -0,0 +1,52 @@
+package sqlitelogger
```

Comment:
I don't think there's anything here related to sqlite or logging? It's implementing an `io.WriteCloser`. Why not in `ee/agent/storage/sqlite/logstore_sqlite.go`?

(I mean, it's also pretty cool, but I'm not sure why it's split out)

Reply:
I see what you mean, I think I originally assumed there would be more to do here. Consolidating should clean up those extra exported types too, I will see how that looks. thank you!